.storybook/main.js

```diff
@@ -0,0 +1,77 @@
+const path = require("path");
+const glob = require('glob');
+const { exec } = require('child_process');
+
+const cssLoader = ["style-loader", "css-loader"];
+const scssLoader = [
+  {
+    loader: "sass-loader",
+    options: {
+      implementation: require("node-sass"),
+    },
+  },
+];
+
+module.exports = {
+  stories: ["../stories/**/*.stories.js"],
+  addons: [
+    "@storybook/addon-actions",
+    "@storybook/addon-links",
+    "@storybook/preset-scss",
+    "@storybook/addon-a11y/register",
+    "@storybook/addon-cssresources/register",
+    "@storybook/addon-viewport/register",
+  ],
+  webpackFinal: async (config, { configType }) => {
+    config.module.rules = [
+      {
+        test: /\.(css|scss)$/,
+        use: [...cssLoader, ...scssLoader],
+        include: path.resolve(__dirname, "../"),
+      },
+      {
+        test: /\.(png|woff|woff2|eot|ttf|svg)$/,
+        loaders: ["file-loader"],
+      },
+      {
+        test: /\.(html|svelte)$/,
+        use: ["svelte-loader"],
+        include: path.resolve(__dirname, "../stories"),
+        exclude: path.resolve(__dirname, "../node_modules"),
+      },
+    ];
+    config.plugins.push({
```

Comment:
Added a watcher for scss changes in the src directory, it will rebuild the compiled css. I did this because we'll want to use storybook in different browsers (it makes ie11 troubleshooting very easy). To make this work, we need to be using the compiled css that has been run through post-css.

pros: 
- we can use storybook to develop our components in safari, firefox, ie11 with the appropriate prefixes

cons: 
- slower

Reply:
I replaced the usage of `../../dist/button.css` to use sass files again because when I run storybook and do some changes on sass file, storybook throws an error because it could not find CSS file (it rebuild the file on the fly and storybook doesn't like it at all). I get the idea and like it :)

Reply:
Weird - it worked fine for me 😿  What was the exact error you got? Had you run `npm run build` before starting storybook? Let's try and troubleshoot it because this same setup works well in another repo we have with storybook

Reply:
I'll try again

Reply:
![css_error](https://user-images.githubusercontent.com/10849982/80695148-c9692600-8ad5-11ea-8528-008fdd868529.gif)

Reply:
Ok, I can't reproduce, but will you make a followup issue to convert this to use the `dist` folder? Otherwise this storybook becomes not very useful for any browser except chrome. Are you using windows by any chance?

Reply:
Yes I'll do, no I'm using linux but I run some tests in Windows VM